projects/simple/tests/Tests/Example/Project.hs

```diff
@@ -1,26 +1,39 @@
 module Tests.Example.Project where
 
-import Prelude
+import Clash.Prelude
+import Clash.Prelude.Testbench
 
 import Test.Tasty
 import Test.Tasty.TH
 import Test.Tasty.Hedgehog
+import Clash.Hedgehog.Sized.Unsigned
+import Clash.Hedgehog.Sized.Vector
 
-import Hedgehog ((===))
-import qualified Hedgehog as H
-import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog as HH
```

Comment:
Did you mean to do this? If anything `H` is a better name when considered in isolation, the animal is a hedgehog, not a hedge hog. Do you habitually have something else under `H` that you don't want Hedgehog there?